src/js/jsx/tools/SuperselectOverlay.jsx

```diff
@@ -327,7 +350,77 @@ define(function (require, exports, module) {
 
             this.updateMouseOverHighlights();
         },
+        
+        /**
+         * Draws either superselectable or leaf layer bounding boxes
+         * 
+         * @param {SVGElement} svg SVG HTML element to draw in
+         * @param {LayerTree} layerTree layerTree of the current document
+         */
```

Comment:
Missing param here... On that note, are you sure you're `npm updated`, because jscs and jshint should complain about this.